<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureEnhancements</class>
 <widget class="QWidget" name="ConfigureEnhancements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>748</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="rendererBox">
     <property name="title">
      <string>Renderer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="stereo_group">
     <property name="title">
      <string>Stereoscopy</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="layout_group">
     <property name="title">
      <string>Layout</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="bg_color_group" native="true">
        <layout class="QHBoxLayout" name="bg_color_group_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="bg_label">
           <property name="text">
            <string>Background Color:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bg_button">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="utilityBox">
     <property name="title">
      <string>Utility</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8"/>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>165</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>bg_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
